src/hotspot/cpu/ppc/macroAssembler_ppc.cpp

```diff
@@ -342,6 +343,27 @@ long MacroAssembler::get_const(address a) {
   }
   return (long) x;
 }
+// Branch-free implementation to convert !0 to false
+// Set register dst to true if dst is non zero using temp for calculations on Power Version<10.
+// Set register dst to true if dst is non zero for Power 10 and above machines.
```

Comment:
Please don't use the term "true". It's not defined in assembler.
In addition: We do the same for older processors. Just by other instructions.
I think these 2 lines could be removed.

Reply:
Can i instead just write 1? I feel the comments can explain the logic in short ?

Reply:
Maybe: "Set register dst to 1 if dst is non-zero. Use setbcr instruction on Power10."